src/librustdoc/formats/cache.rs

```diff
@@ -105,7 +107,8 @@ pub(crate) struct Cache {
     // then the fully qualified name of the structure isn't presented in `paths`
     // yet when its implementation methods are being indexed. Caches such methods
     // and their parent id here and indexes them at the end of crate parsing.
-    pub(crate) orphan_impl_items: Vec<(DefId, clean::Item)>,
+    pub(crate) orphan_impl_items:
+        Vec<(DefId, clean::Item, Option<(clean::Type, clean::Generics)>, bool)>,
```

Comment:
Please either extract a struct with named fields for this tuple (preferred)
or add docs explaining what the bool is.